src/lib/components/termsAndConditions/termsAndConditions.js

```diff
@@ -16,14 +18,16 @@ class TermsAndConditions extends PureComponent {
       drive,
       children,
       checkboxLabel,
-      backgroundColor,
+      // backgroundColor,
       className,
       containerClassName,
       value,
+      // paragraph,
       height,
     } = this.props;
 
-    const containerStyle = { backgroundColor };
+    // const containerStyle = { backgroundColor };
```

Comment:
I removed it from here because it was outputting the background-color inline and I thought we could instead assign the background-color in the CSS. I didn't want to delete entirely until I asked @kbardi why it was this way (in case I'm missing something 🙊). Buuuttt, I don't think this is what was breaking it - I rather wanted help from @kbardi for putting a component within a component. Eg. `<Paragraph>` (instead of `<p>`) within `<TermsAndConditions>` 😝

Reply:
@lulen11 if you run `yarn deploy` you'll see the tests don't pass but I'll leave you and @kbardi to work through that 👍🏼

Reply:
I created a method to update snapshots...so when we change a component we should update snapshots to make the build work well...you can use `yarn test:update-snapshot`